af/shovel/centrifugation.py

```diff
@@ -34,6 +34,10 @@
 from oonipl.sim import sim_shi4_mm3_layout, sim_shi4_mm3_text
 from oonipl.utils import dclass
 
+from oonipl.metrics import setup_metrics
+
+metrics = setup_metrics(host="172.17.0.1", name="centrifugation")
```

Comment:
I believe we should probably document what this IP address is. I cannot figure out immediately what is its purpose, but I believe the `172.17` space is used by Docker, right?

Reply:
Updated